ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -2129,7 +2129,7 @@ def test_numpy_instance_int__(
     init_tree="numpy.array",
     method_name="__float__",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("numeric"),
```

Comment:
Test cases fail because of incompatible dtypes coming up in `numeric`

Reply:
I think you should leave this one as it is. Numeric seems reasonable to me. Plus, it's unrelated to this task as far as I can tell and also could be left failing deliberately by someone else to be fixed later.